Missing Peer Dependencies and Type Declarations in @citolab/qti-components

When installing @citolab/qti-components via npm and using it in an Angular project, the build fails with multiple TypeScript errors related to missing modules such as lit, lit-html, and @heximal/templates. These errors indicate that the package has peer dependencies which are not automatically installed, and some type declarations may be missing or incompatible.

**Example errors:**
TS2307: Cannot find module 'lit' or its corresponding type declarations.
TS2307: Cannot find module 'lit-html' or its corresponding type declarations.
TS2307: Cannot find module '@heximal/templates' or its corresponding type declarations.

**Impact:**
Developers must manually install these dependencies and possibly add type declarations, which is not documented and can lead to confusion and integration issues.
This breaks the out-of-the-box experience and may cause compatibility problems with Angular’s strict type checking.

--- a/src/lib/qti-test/components/test-scoring-feedback.ts
+++ b/src/lib/qti-test/components/test-scoring-feedback.ts
@@ -4,9 +4,9 @@ import { customElement, property } from 'lit/decorators.js';
 
 import { computedContext } from '../../exports/computed.context';
 
+import type { View } from '../../exports/session.context';
 import type { OutcomeVariable } from '../../exports/variables';
 import type { ComputedContext } from '../../exports/computed.context';
-import type { ViewMode } from 'storybook/internal/types';
 
 @customElement('test-scoring-feedback')
 export class TestScoringFeedback extends LitElement {
@@ -14,7 +14,7 @@ export class TestScoringFeedback extends LitElement {
   protected computedContext?: ComputedContext;
 
   @property({ type: String, attribute: 'view' })
-  public view: ViewMode = null;
+  public view: View = null;
 
   render() {
     const activeItem = this.computedContext?.testParts

